.github/workflows/go.yml

```diff
@@ -0,0 +1,46 @@
+name: Push
+
+on:
+  push:
+    branches: [ master ]
+    tags:
+      - '*'
+
+jobs:
+
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go 1.14
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.14
+      id: go
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+
+    - name: Get dependencies
+      run: |
+        go get -v -t -d ./...
+        if [ -f Gopkg.toml ]; then
+            curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
+            dep ensure
+        fi
+
+    - name: Build
+      run: make all
+
+    - name: Test
+      run: make test
+    
+    - name: Get Tag
+      id: get_tag
+      run: |
+        echo ::set-output name=TAG::$(echo $GITHUB_REF | cut -d / -f 3)
+        
+    - name: Publish container image
+      run: |
+        TAG=${{ steps.get_tag.outputs.TAG }} make container
```

Comment:
this should be `make push`, right?

Also, need to figure out how to provide the docker hub credentials.

Reply:
Yes. That has to be a `push`. Added that target to the Makefile. Wanted to get this to work before pushing random images to the registry from my fork.